src/main/kotlin/no/nav/familie/ks/sak/barnehagelister/BarnehageListeService.kt

```diff
@@ -98,25 +101,81 @@ class BarnehageListeService(
         return barnehagelisteMottattRepository.findAllIds()
     }
 
-    fun hentAlleBarnehagebarnPage(barnehagebarnRequestParams: BarnehagebarnRequestParams): Page<BarnehagebarnDtoInterface> {
+    fun hentAlleBarnehagebarnInfotrygd(barnehagebarnRequestParams: BarnehagebarnRequestParams): Page<BarnehagebarnInfotrygdDto> {
         var sort = Sort.by(getCorrectSortBy("kommuneNavn")).descending()
-        var fagsakstatuser =
+
+        if (barnehagebarnRequestParams.sortAsc) {
+            sort = Sort.by(getCorrectSortBy(barnehagebarnRequestParams.sortBy)).ascending()
+        } else {
+            sort = Sort.by(getCorrectSortBy(barnehagebarnRequestParams.sortBy)).descending()
+        }
+
+        val pageable: Pageable =
+            PageRequest.of(barnehagebarnRequestParams.offset, barnehagebarnRequestParams.limit, sort)
+
+        val barna = infotrygdReplikaClient.hentAlleBarnasIdenterForLøpendeFagsaker()
```

Comment:
Caching?

Reply:
Vi ser hvordan det går uten først, også legger vi det på hvis det er behov for det.